generators/plugin/templates/sample_bastion_plugin.gemspec

```diff
@@ -0,0 +1,20 @@
+$:.push File.expand_path("../lib", __FILE__)
+
+# Maintain your gem's version:
+require "<%= nameSnake %>/version"
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = "<%= nameSnake %>"
+  s.version     = <%= namePascal %>::VERSION
+  s.authors     = ["thomasfmckay@gmail.com"]
```

Comment:
Noted. I will make the final gemspec be either fully configurable via yo or leave as defaults that the plugin creator will need to fill in.